samples/boards/nrf52/mesh/onoff_level_vnd_app/prj.conf

```diff
@@ -0,0 +1,81 @@
+CONFIG_INIT_STACKS=n
+CONFIG_MAIN_STACK_SIZE=512
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+CONFIG_BT_SETTINGS=y
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_FLASH_MAP=y
+CONFIG_FCB=y
+CONFIG_SETTINGS=y
+CONFIG_SETTINGS_FCB=y
+CONFIG_BT_MESH_RPL_STORE_TIMEOUT=5
+
+CONFIG_BT_OBSERVER=y
+CONFIG_BT_BROADCASTER=y
+CONFIG_BT_CTLR_DUP_FILTER_LEN=0
+CONFIG_BT_CTLR_LE_ENC=n
+CONFIG_BT_CTLR_LE_PING=n
+CONFIG_BT_CTLR_DATA_LENGTH=n
+CONFIG_BT_CTLR_PHY=n
+CONFIG_BT_CTLR_CHAN_SEL_2=n
+CONFIG_BT_CTLR_MIN_USED_CHAN=n
+CONFIG_BT_CTLR_ADV_EXT=n
+CONFIG_BT_CTLR_PRIVACY=n
+
+CONFIG_BT_CTLR_TX_PWR_PLUS_8=y
+
+CONFIG_BT_PERIPHERAL=y
+
+CONFIG_BT=y
+CONFIG_BT_TINYCRYPT_ECC=y
+CONFIG_BT_RX_BUF_COUNT=30
+CONFIG_BT_L2CAP_RX_MTU=69
+CONFIG_BT_L2CAP_TX_MTU=69
+CONFIG_BT_L2CAP_TX_BUF_COUNT=8
+
+CONFIG_BT_MESH=y
+CONFIG_BT_MESH_RELAY=y
+CONFIG_BT_MESH_LOW_POWER=n
+CONFIG_BT_MESH_FRIEND=n
+CONFIG_BT_MESH_FRIEND_QUEUE_SIZE=16
+CONFIG_BT_MESH_ADV_BUF_COUNT=20
+
+CONFIG_BT_MESH_PB_GATT=y
+CONFIG_BT_MESH_PB_ADV=y
+CONFIG_BT_MESH_GATT_PROXY=y
+
+##CONFIG_BT_MESH_LPN_SCAN_LATENCY=30
+##CONFIG_BT_MESH_LPN_RECV_DELAY=40
+#CONFIG_BT_MESH_LPN_POLL_TIMEOUT=300
+
+CONFIG_BT_MESH_SUBNET_COUNT=2
+CONFIG_BT_MESH_APP_KEY_COUNT=2
+CONFIG_BT_MESH_MODEL_GROUP_COUNT=2
+CONFIG_BT_MESH_LABEL_COUNT=3
+
+#CONFIG_BT_MESH_IV_UPDATE_TEST=y
+
+#CONFIG_UART_CONSOLE=n
+#CONFIG_BT_DEBUG_MONITOR=y
+##CONFIG_BT_DEBUG_LOG=y
+
+##CONFIG_BT_MESH_DEBUG=y
+#CONFIG_BT_MESH_DEBUG_PROV=y
+#CONFIG_BT_MESH_DEBUG_PROXY=y
+#CONFIG_BT_MESH_DEBUG_BEACON=y
+##CONFIG_BT_MESH_DEBUG_NET=y
+##CONFIG_BT_MESH_DEBUG_TRANS=y
+##CONFIG_BT_MESH_DEBUG_SETTINGS=y
+#CONFIG_BT_MESH_DEBUG_LOW_POWER=y
+##CONFIG_BT_MESH_DEBUG_FRIEND=y
+#CONFIG_BT_MESH_DEBUG_MODEL=y
+#CONFIG_BT_MESH_DEBUG_ACCESS=y
+#CONFIG_BT_MESH_DEBUG_CRYPTO=y
+#CONFIG_BT_MESH_DEBUG_ADV=y
+#CONFIG_BT_MESH_SELF_TEST=y
+
+CONFIG_BT_RX_STACK_SIZE=4096
+
+
```

Comment:
Redundant empty line here (there should ever be at most one)

Reply:
removed !!